app/schema.graphql

```diff
@@ -41,6 +68,13 @@ type AnnotationSummary {
   labelCount: Int!
 }
 
+enum AnnotationType {
+  CATEGORICAL
+  CONTINUOUS
+  FREEFORM
+  BINARY
```

Comment:
I would nix? It's never benefited me to make this distinction - it just pollutes the data model

Reply:
yes, let's nix binary